src/meeting.ts

```
@@ -10,41 +9,30 @@ import {
 
 export class MeetingHandle {
     static instance: MeetingHandle = null
     public stateMachine: MeetingStateMachine
-    private param: MeetingParams
     private provider: MeetingProviderInterface
 
-    static init(meetingParams: MeetingParams) {
+    static init() {
         if (MeetingHandle.instance == null) {
-            this.instance = new MeetingHandle(meetingParams)
+            this.instance = new MeetingHandle()
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)